devito/core/operator.py

```diff
@@ -67,6 +69,11 @@ class BasicOperator(Operator):
     intensity of the generated kernel.
     """
 
+    SCALAR_MIN_TYPE = np.float16
+    """
+    Minimum datatype for a scalar alias for a common sub-expression or CIRE temp.
```

Comment:
"alias" applies just to CIRE. So more correct would be "... for a scalar arising from ..."